### PR TITLE
Added to readme

### PR DESCRIPTION
**Pull request by Jambls**
Let me know if there's anything else that would be good to add.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,13 @@ To build just the vocoder itself you will need:
     - Debian
     - ALSA
 - Hardware
-    - Raspberry Pi 3 (or newer)
+    - Raspberry Pi 3 (or newer) running a 64bit OS
     - Codec Zero
 
 Additionally, to run the unit tests and static analysis suite, LLVM 15 + Tools is required. To install this on debian the automatic script from [their website](https://apt.llvm.org/) can be used.
 
+To build the doxygen documentation, the `doxygen` and `graphviz` packages are required.
```
Comment (MHCooke) [inline README.md:20]: I think it looks messy to have these in a paragraph on their own. I think they should be added to the block of commands below. We could also make an additional list of optional software requirements in the list above.

File: README.md
Comment (MHCooke) [inline README.md:1]: We should probably add a description of the license and how to comply with it to the readme

File: README.md (hunk 1 above)
Comment (Jambls) [reply README.md:20]: It matches the paragraph above.

Comment (Jambls) [reply README.md:20]: Added a bit to the docs section that explains installing them.

File: README.md [hunk 2]
```diff
@@ -64,24 +64,49 @@ chmod +x checks.sh
...
+
+# Project management
+
+Regualar scrum meetings were held through the project, notes are available [here](https://github.com/daft-engineers/project/blob/main/README.md). 
+
+The project uses github's issue system for management. The project board is located [here.](https://github.com/orgs/daft-engineers/projects/1/views/3) ![Image](https://user-images.githubusercontent.com/56744932/232234904-d9e0ba0a-6532-460d-966f-4a65a5f11cc9.png)
+
+# Contributions 
+
+Contributions must be made in the form of a pull request and must pass the unit tests and validation suite. Any contrbutions must be licenceable under the 2.0.
```
Comment (MHCooke) [inline README.md:102]:
```suggestion
Contributions must be made in the form of a pull request and must pass the unit tests and validation suite. Any contrbutions must be licenceable under the MPL 2.0.
```

File: README.md
Comment (MHCooke) [reply README.md:1]: Maybe add a link to this? https://choosealicense.com/licenses/mpl-2.0/

File: README.md [hunk 3]
```diff
@@ -64,24 +64,49 @@ chmod +x checks.sh
...
 # Documentation
 
-Documentation for main can be [found here](https://daft-engineers.github.io/vocoder/).
+Documentation for main can be [found here](https://daft-engineers.github.io/vocoder/annotated.html).
+
+If you want to generate the documentation for yourself, the `doxygen` and `graphviz` packages are required. These can be installed with the following command on ubuntu:
 
-If you want to generate the documentation for yourself, from the root of the project, the command 
+```$
+sudo apt install doxygen graphviz
```
Comment (bear-in-the-air) [inline README.md:74]: Can you not just run "doxygen" from the project root?

File: README.md [hunk 4]
```diff
@@ -12,11 +12,13 @@ To build just the vocoder itself you will need:
     - Debian
     - ALSA
 - Hardware
-    - Raspberry Pi 3 (or newer)
+    - Raspberry Pi 3 (or newer) running a 64bit OS
     - Codec Zero
```
Comment (bear-in-the-air) [inline README.md:16]: I think the hardware isn't necessarily a codec zero, just that the audio comes in on the correct channels and alsa sets the inputs and outputs we need.

I don't think we ever talk about setting up the codec zero anywyhere either

File: README.md (hunk 3 above)
Comment (Jambls) [reply README.md:74]: if you've got it installed, yeah

File: README.md (hunk 4 above)
Comment (MHCooke) [reply README.md:16]: However, this is written to work on a codec zero. Use something else at your own risk.

Comment (Jambls) [reply README.md:16]: It won't work on other hardware without changing the alsa setup, so I'd say the codec zero is required. 

You're right about the setup though, however, we don't know exactly what the final setup will be at the moment, so I'd say it's better to leave it out than to have a conspicuous section.

File: README.md (hunk 3 above)
Comment (MHCooke) [reply README.md:74]: He means you don't need to do `doxygen Doxyfile`, but it doesn't matter imo. It works as is

File: README.md (hunk 4 above)
Comment (bear-in-the-air) [reply README.md:16]: Ok, it can be left for now

**Review by bear-in-the-air — APPROVED** [review]

**Review by MHCooke — APPROVED** [review]
Looks good

**Pull request merged**